src/combinatorics/simple_graph/basic.lean

```diff
@@ -906,9 +905,22 @@ protected def map (f : V ↪ W) (G : simple_graph V) : simple_graph W :=
 @[simp] lemma map_adj (f : V ↪ W) (G : simple_graph V) (u v : W) :
   (G.map f).adj u v ↔ ∃ (u' v' : V), G.adj u' v' ∧ f u' = u ∧ f v' = v := iff.rfl
 
+lemma map_adj_apply {G : simple_graph V} {f : V ↪ W} {a b : V} :
+  (G.map f).adj (f a) (f b) ↔ G.adj a b := by simp
+
 lemma map_monotone (f : V ↪ W) : monotone (simple_graph.map f) :=
 by { rintros G G' h _ _ ⟨u, v, ha, rfl, rfl⟩, exact ⟨_, _, h ha, rfl, rfl⟩ }
 
+@[simp] lemma map_id : G.map (function.embedding.refl _) = G :=
```

Comment:
```suggestion
@[simp] lemma map_refl : G.map (function.embedding.refl _) = G :=
```
maybe

Reply:
`function.embedding.refl` is such a bad name...

Reply:
Yeah no, `map_refl` will make this lemma undiscoverable.